src/Microsoft.TestPlatform.Utilities/InferRunSettingsHelper.cs

```diff
@@ -220,7 +242,7 @@ public static bool TryGetDeviceXml(XPathNavigator runSettingsNavigator, out Stri
         /// <summary>
         /// Adds node under RunConfiguration setting. Noop if node is already present.
         /// </summary>
-        private static void AddNodeIfNotPresent<T>(XPathNavigator runSettingsNavigator, string nodePath, string nodeName, T nodeValue)
+        private static void AddNodeIfNotPresent<T>(XPathNavigator runSettingsNavigator, string nodePath, string nodeName, T nodeValue, bool overwrite = false)
```

Comment:
What is the need for adding `bool overwrite`?

Reply:
Required this to overwrite default settings.